src/styles/molecules/_toast.scss

```diff
@@ -35,13 +35,20 @@
     width: 100%;
     padding: $dc-space100 0;
     border-width: 1px;
-    border-style: solid;
+    border-style: none;
     border-radius: $dc-radius50;
     font-size: $dc-font50;
     font-weight: $dc-bold-font-weight;
     line-height: 1.2;
     text-align: center;
     text-transform: uppercase;
+
+    &--success,
```

Comment:
I think this breaks a little bit the concept of not relying of class names and give the possibility to the end user to use mixins to compose the style of a component. @sambhav-gore WDYT?